{
    "inputs": {
        "dependency": "tensorflow",
        "outfile": "/home/vanessa/Desktop/Code/caliper-analysis/.caliper/pypi-tensorflow-2.0.0a0-python-cp27.json",
        "dockerfile": "FROM python:2.7\nADD . /tmp/repo\nWORKDIR /tmp/repo\nENV LANG C.UTF-8\nENV SHELL /bin/bash\nRUN apt-get update && apt-get install -y wget ca-certificates gnupg2 git\nRUN /bin/bash -c \"wget https://files.pythonhosted.org/packages/bc/84/970bfb6eb04360a675627a38962127f0f5302ac1fd3ac4ad4f5d1befc9b7/tensorflow-2.0.0a0-cp27-cp27mu-manylinux1_x86_64.whl && pip install tensorflow-2.0.0a0-cp27-cp27mu-manylinux1_x86_64.whl\"\n",
        "force": false,
        "exists": true,
        "name": "pypi-tensorflow-2.0.0a0-python-cp27",
        "tests": "tensorflow_v0.11/5_MultiGPU/multigpu_basics.py\ntensorflow_v0.11/1_Introduction/basic_operations.py\ntensorflow_v0.11/1_Introduction/helloworld.py\ntensorflow_v0.11/4_Utils/tensorboard_advanced.py\ntensorflow_v0.11/4_Utils/tensorboard_basic.py\ntensorflow_v0.11/2_BasicModels/logistic_regression.py\ntensorflow_v0.11/2_BasicModels/linear_regression.py\ntensorflow_v0.11/2_BasicModels/nearest_neighbor.py\ntensorflow_v0.11/3_NeuralNetworks/bidirectional_rnn.py\ntensorflow_v0.11/3_NeuralNetworks/autoencoder.py\ntensorflow_v0.11/3_NeuralNetworks/recurrent_network.py\ntensorflow_v0.11/3_NeuralNetworks/dynamic_rnn.py\ntensorflow_v0.11/3_NeuralNetworks/multilayer_perceptron.py\ntensorflow_v0.11/3_NeuralNetworks/convolutional_network.py\ntensorflow_v1/1_Introduction/basic_eager_api.py\ntensorflow_v1/1_Introduction/basic_operations.py\ntensorflow_v1/1_Introduction/helloworld.py\ntensorflow_v1/4_Utils/tensorboard_advanced.py\ntensorflow_v1/4_Utils/tensorboard_basic.py\ntensorflow_v1/5_DataManagement/build_an_image_dataset.py\ntensorflow_v1/5_DataManagement/tensorflow_dataset_api.py\ntensorflow_v1/2_BasicModels/random_forest.py\ntensorflow_v1/2_BasicModels/gradient_boosted_decision_tree.py\ntensorflow_v1/2_BasicModels/logistic_regression.py\ntensorflow_v1/2_BasicModels/word2vec.py\ntensorflow_v1/2_BasicModels/linear_regression.py\ntensorflow_v1/2_BasicModels/logistic_regression_eager_api.py\ntensorflow_v1/2_BasicModels/nearest_neighbor.py\ntensorflow_v1/2_BasicModels/linear_regression_eager_api.py\ntensorflow_v1/2_BasicModels/kmeans.py\ntensorflow_v1/6_MultiGPU/multigpu_basics.py\ntensorflow_v1/6_MultiGPU/multigpu_cnn.py\ntensorflow_v1/3_NeuralNetworks/variational_autoencoder.py\ntensorflow_v1/3_NeuralNetworks/neural_network_eager_api.py\ntensorflow_v1/3_NeuralNetworks/neural_network.py\ntensorflow_v1/3_NeuralNetworks/bidirectional_rnn.py\ntensorflow_v1/3_NeuralNetworks/convolutional_network_raw.py\ntensorflow_v1/3_NeuralNetworks/autoencoder.py\ntensorflow_v1/3_NeuralNetworks/gan.py\ntensorflow_v1/3_NeuralNetworks/recurrent_network.py\ntensorflow_v1/3_NeuralNetworks/neural_network_raw.py\ntensorflow_v1/3_NeuralNetworks/multilayer_perceptron.py\ntensorflow_v1/3_NeuralNetworks/convolutional_network.py",
        "cleanup": true,
        "outdir": "/home/vanessa/Desktop/Code/caliper-analysis"
    },
    "tests": {
        "build": {
            "retval": 0
        },
        "tensorflow_v0.11/5_MultiGPU/multigpu_basics.py": {
            "error": [
                "2021-01-04 22:11:06.698174: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-04 22:11:06.729633: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-04 22:11:06.730063: I tensorflow/compiler/xla/service/service.cc:162] XLA service 0x55bbb24c6b40 executing computations on platform Host. Devices:\n",
                "2021-01-04 22:11:06.730096: I tensorflow/compiler/xla/service/service.cc:169]   StreamExecutor device (0): <undefined>, <undefined>\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/5_MultiGPU/multigpu_basics.py\", line 61, in <module>\n",
                "    a = tf.placeholder(tf.float32, [1000, 1000])\n",
                "AttributeError: 'module' object has no attribute 'placeholder'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.92
        },
        "tensorflow_v0.11/1_Introduction/basic_operations.py": {
            "error": [
                "2021-01-04 22:11:08.831092: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-04 22:11:08.877585: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-04 22:11:08.878049: I tensorflow/compiler/xla/service/service.cc:162] XLA service 0x55589e758010 executing computations on platform Host. Devices:\n",
                "2021-01-04 22:11:08.878074: I tensorflow/compiler/xla/service/service.cc:169]   StreamExecutor device (0): <undefined>, <undefined>\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/1_Introduction/basic_operations.py\", line 25, in <module>\n",
                "    with tf.Session() as sess:\n",
                "AttributeError: 'module' object has no attribute 'Session'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.1
        },
        "tensorflow_v0.11/1_Introduction/helloworld.py": {
            "error": [
                "2021-01-04 22:11:10.734825: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-04 22:11:10.769649: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-04 22:11:10.770162: I tensorflow/compiler/xla/service/service.cc:162] XLA service 0x5572326ad1a0 executing computations on platform Host. Devices:\n",
                "2021-01-04 22:11:10.770190: I tensorflow/compiler/xla/service/service.cc:169]   StreamExecutor device (0): <undefined>, <undefined>\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/1_Introduction/helloworld.py\", line 28, in <module>\n",
                "    sess = tf.Session()\n",
                "AttributeError: 'module' object has no attribute 'Session'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.88
        },
        "tensorflow_v0.11/4_Utils/tensorboard_advanced.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/4_Utils/tensorboard_advanced.py\", line 21, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.01
        },
        "tensorflow_v0.11/4_Utils/tensorboard_basic.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/4_Utils/tensorboard_basic.py\", line 21, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.01
        },
        "tensorflow_v0.11/2_BasicModels/logistic_regression.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/2_BasicModels/logistic_regression.py\", line 21, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.91
        },
        "tensorflow_v0.11/2_BasicModels/linear_regression.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/2_BasicModels/linear_regression.py\", line 73, in <module>\n",
                "    X = tf.placeholder(\"float\")\n",
                "AttributeError: 'module' object has no attribute 'placeholder'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.15
        },
        "tensorflow_v0.11/2_BasicModels/nearest_neighbor.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/2_BasicModels/nearest_neighbor.py\", line 23, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.81
        },
        "tensorflow_v0.11/3_NeuralNetworks/bidirectional_rnn.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/bidirectional_rnn.py\", line 24, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.97
        },
        "tensorflow_v0.11/3_NeuralNetworks/autoencoder.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/autoencoder.py\", line 25, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.07
        },
        "tensorflow_v0.11/3_NeuralNetworks/recurrent_network.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/recurrent_network.py\", line 22, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.87
        },
        "tensorflow_v0.11/3_NeuralNetworks/dynamic_rnn.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/dynamic_rnn.py\", line 107, in <module>\n",
                "    x = tf.placeholder(\"float\", [None, seq_max_len, 1])\n",
                "AttributeError: 'module' object has no attribute 'placeholder'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.19
        },
        "tensorflow_v0.11/3_NeuralNetworks/multilayer_perceptron.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/multilayer_perceptron.py\", line 13, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.83
        },
        "tensorflow_v0.11/3_NeuralNetworks/convolutional_network.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/convolutional_network.py\", line 21, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.97
        },
        "tensorflow_v1/1_Introduction/basic_eager_api.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/1_Introduction/basic_eager_api.py\", line 23, in <module>\n",
                "    import tensorflow.contrib.eager as tfe\n",
                "ImportError: No module named contrib.eager\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.16
        },
        "tensorflow_v1/1_Introduction/basic_operations.py": {
            "error": [
                "2021-01-04 22:11:36.556007: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-04 22:11:36.589635: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-04 22:11:36.590154: I tensorflow/compiler/xla/service/service.cc:162] XLA service 0x5592a15b3c70 executing computations on platform Host. Devices:\n",
                "2021-01-04 22:11:36.590198: I tensorflow/compiler/xla/service/service.cc:169]   StreamExecutor device (0): <undefined>, <undefined>\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/1_Introduction/basic_operations.py\", line 25, in <module>\n",
                "    with tf.Session() as sess:\n",
                "AttributeError: 'module' object has no attribute 'Session'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.91
        },
        "tensorflow_v1/1_Introduction/helloworld.py": {
            "error": [
                "2021-01-04 22:11:38.597731: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-04 22:11:38.645628: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-04 22:11:38.648849: I tensorflow/compiler/xla/service/service.cc:162] XLA service 0x55b83f6bc080 executing computations on platform Host. Devices:\n",
                "2021-01-04 22:11:38.648893: I tensorflow/compiler/xla/service/service.cc:169]   StreamExecutor device (0): <undefined>, <undefined>\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/1_Introduction/helloworld.py\", line 28, in <module>\n",
                "    sess = tf.Session()\n",
                "AttributeError: 'module' object has no attribute 'Session'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.24
        },
        "tensorflow_v1/4_Utils/tensorboard_advanced.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/4_Utils/tensorboard_advanced.py\", line 20, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.9
        },
        "tensorflow_v1/4_Utils/tensorboard_basic.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/4_Utils/tensorboard_basic.py\", line 20, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.08
        },
        "tensorflow_v1/5_DataManagement/build_an_image_dataset.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/5_DataManagement/build_an_image_dataset.py\", line 133, in <module>\n",
                "    X, Y = read_images(DATASET_PATH, MODE, batch_size)\n",
                "  File \"tensorflow_v1/5_DataManagement/build_an_image_dataset.py\", line 76, in read_images\n",
                "    classes = sorted(os.walk(dataset_path).__next__()[1])\n",
                "AttributeError: 'generator' object has no attribute '__next__'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.93
        },
        "tensorflow_v1/5_DataManagement/tensorflow_dataset_api.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/5_DataManagement/tensorflow_dataset_api.py\", line 20, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.84
        },
        "tensorflow_v1/2_BasicModels/random_forest.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/random_forest.py\", line 14, in <module>\n",
                "    from tensorflow.contrib.tensor_forest.python import tensor_forest\n",
                "ImportError: No module named contrib.tensor_forest.python\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.13
        },
        "tensorflow_v1/2_BasicModels/gradient_boosted_decision_tree.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/gradient_boosted_decision_tree.py\", line 17, in <module>\n",
                "    from tensorflow.contrib.boosted_trees.estimator_batch.estimator import (\n",
                "ImportError: No module named contrib.boosted_trees.estimator_batch.estimator\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.86
        },
        "tensorflow_v1/2_BasicModels/logistic_regression.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/logistic_regression.py\", line 21, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.94
        },
        "tensorflow_v1/2_BasicModels/word2vec.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/word2vec.py\", line 131, in <module>\n",
                "    X = tf.placeholder(tf.int32, shape=[None])\n",
                "AttributeError: 'module' object has no attribute 'placeholder'\n"
            ],
            "output": [
                "Downloading the dataset... (It may take some time)\n",
                "Done!\n",
                "Words count: 17005207\n",
                "Unique words: 253854\n",
                "Vocabulary size: 47135\n",
                "Most common words: [('UNK', 444176), ('the', 1061396), ('of', 593677), ('and', 416629), ('one', 411764), ('in', 372201), ('a', 325873), ('to', 316376), ('zero', 264975), ('nine', 250430)]\n"
            ],
            "retval": 1,
            "seconds": 66.83
        },
        "tensorflow_v1/2_BasicModels/linear_regression.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/linear_regression.py\", line 73, in <module>\n",
                "    X = tf.placeholder(\"float\")\n",
                "AttributeError: 'module' object has no attribute 'placeholder'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.96
        },
        "tensorflow_v1/2_BasicModels/logistic_regression_eager_api.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/logistic_regression_eager_api.py\", line 20, in <module>\n",
                "    tf.enable_eager_execution()\n",
                "AttributeError: 'module' object has no attribute 'enable_eager_execution'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.95
        },
        "tensorflow_v1/2_BasicModels/nearest_neighbor.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/nearest_neighbor.py\", line 23, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.2
        },
        "tensorflow_v1/2_BasicModels/linear_regression_eager_api.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/linear_regression_eager_api.py\", line 21, in <module>\n",
                "    tf.enable_eager_execution()\n",
                "AttributeError: 'module' object has no attribute 'enable_eager_execution'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.04
        },
        "tensorflow_v1/2_BasicModels/kmeans.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/kmeans.py\", line 17, in <module>\n",
                "    from tensorflow.contrib.factorization import KMeans\n",
                "ImportError: No module named contrib.factorization\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.03
        },
        "tensorflow_v1/6_MultiGPU/multigpu_basics.py": {
            "error": [
                "2021-01-04 22:13:11.930563: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-04 22:13:11.965651: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-04 22:13:11.966256: I tensorflow/compiler/xla/service/service.cc:162] XLA service 0x555d76346f90 executing computations on platform Host. Devices:\n",
                "2021-01-04 22:13:11.966322: I tensorflow/compiler/xla/service/service.cc:169]   StreamExecutor device (0): <undefined>, <undefined>\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/6_MultiGPU/multigpu_basics.py\", line 61, in <module>\n",
                "    a = tf.placeholder(tf.float32, [1000, 1000])\n",
                "AttributeError: 'module' object has no attribute 'placeholder'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.49
        },
        "tensorflow_v1/6_MultiGPU/multigpu_cnn.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/6_MultiGPU/multigpu_cnn.py\", line 28, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.91
        },
        "tensorflow_v1/3_NeuralNetworks/variational_autoencoder.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/variational_autoencoder.py\", line 35, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.28
        },
        "tensorflow_v1/3_NeuralNetworks/neural_network_eager_api.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/neural_network_eager_api.py\", line 26, in <module>\n",
                "    tf.enable_eager_execution()\n",
                "AttributeError: 'module' object has no attribute 'enable_eager_execution'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.98
        },
        "tensorflow_v1/3_NeuralNetworks/neural_network.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/neural_network.py\", line 20, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.07
        },
        "tensorflow_v1/3_NeuralNetworks/bidirectional_rnn.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/bidirectional_rnn.py\", line 18, in <module>\n",
                "    from tensorflow.contrib import rnn\n",
                "ImportError: No module named contrib\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.28
        },
        "tensorflow_v1/3_NeuralNetworks/convolutional_network_raw.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/convolutional_network_raw.py\", line 16, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.05
        },
        "tensorflow_v1/3_NeuralNetworks/autoencoder.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/autoencoder.py\", line 30, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.22
        },
        "tensorflow_v1/3_NeuralNetworks/gan.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/gan.py\", line 35, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.94
        },
        "tensorflow_v1/3_NeuralNetworks/recurrent_network.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/recurrent_network.py\", line 17, in <module>\n",
                "    from tensorflow.contrib import rnn\n",
                "ImportError: No module named contrib\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.05
        },
        "tensorflow_v1/3_NeuralNetworks/neural_network_raw.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/neural_network_raw.py\", line 17, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.36
        },
        "tensorflow_v1/3_NeuralNetworks/multilayer_perceptron.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/multilayer_perceptron.py\", line 24, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 1.98
        },
        "tensorflow_v1/3_NeuralNetworks/convolutional_network.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/convolutional_network.py\", line 16, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ImportError: No module named examples.tutorials.mnist\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.0
        }
    },
    "requirements.txt": [
        "absl-py==0.11.0\n",
        "appdirs==1.4.3\n",
        "astor==0.8.1\n",
        "backports.weakref==1.0.post1\n",
        "configparser==4.0.2\n",
        "contextlib2==0.6.0.post1\n",
        "distlib==0.3.0\n",
        "enum34==1.1.10\n",
        "filelock==3.0.12\n",
        "funcsigs==1.0.2\n",
        "futures==3.3.0\n",
        "gast==0.4.0\n",
        "google-pasta==0.2.0\n",
        "grpcio==1.34.0\n",
        "h5py==2.10.0\n",
        "importlib-metadata==1.6.0\n",
        "importlib-resources==1.4.0\n",
        "Keras-Applications==1.0.8\n",
        "Keras-Preprocessing==1.1.2\n",
        "Markdown==3.1.1\n",
        "mock==3.0.5\n",
        "numpy==1.16.6\n",
        "pathlib2==2.3.5\n",
        "protobuf==3.14.0\n",
        "scandir==1.10.0\n",
        "singledispatch==3.4.0.3\n",
        "six==1.14.0\n",
        "tb-nightly==1.14.0a20190301\n",
        "tensorflow==2.0.0a0\n",
        "termcolor==1.1.0\n",
        "tf-estimator-nightly==1.14.0.dev2019030115\n",
        "typing==3.7.4.1\n",
        "virtualenv==20.0.18\n",
        "Werkzeug==1.0.1\n",
        "zipp==1.2.0\n"
    ]
}